### ISSUE TITLE
Have a procedure to add code in the project

### ISSUE THREAD

**Issue by hehrhart** [Supermood/dev-handbook#6]
__PROPOSAL__: Create a `CONTRIBUTING.md` file in each repository and explain with a shema and images in the handbook.

**Comment by hilnius** [comment]
@hehrhart to clarify, you mean that each repo should host a "contributing.md" file at the root, is that correct ?

**Comment by hehrhart** [comment]
Yes, each repository must contain a `CONTRIBUTING.md` with a link to the company handbook and the specific rules to apply to this project. It can be another CI with label / tag which are in addition.
